lib/imgix/client.rb

```diff
@@ -60,6 +52,27 @@ def prefix
 
     private
 
+    def create_request(endpoint, img_url)
+      req = Net::HTTP::Post.new(endpoint.path)
+      req["Content-Type"] = "application/json"
```

Comment:
You may have to check with BT, but I think the `Content-Type` will have to be `application/vnd.api+json` by default for purge requests.
https://github.com/imgix/imgix-management-js/blob/e3ed4cd2c4414eb816dad515794bae581c76860f/src/imgix-api.js#L59